src/itree.cpp

```diff
@@ -132,15 +160,32 @@ const char* permission_type_string(fs::perms p) {
     }
 }
 
+namespace {
```

Comment:
For consistency, let's put the open curly brace on its own line.

Or remove the namespace entirely since this is specific to the itree.cpp file.

Reply:
done